### ISSUE TITLE
Collection view based rows aren't scrollable using VoiceOver

### ISSUE THREAD

**Issue by simonmitchell** [3sidedcube/ThunderCloud#205]
## Expected Behaviour

The user should be able to use 3-finger swipe or swipe to navigate through items in a horizontally scrollable list item using VoiceOver.

## Current Behaviour

Only the first item in collection view based list items is reachable

## Steps to Reproduce (for bugs)

1. Create a content page with a spotlight or app collection e.t.c.
2. Enable VoiceOver on the test device
3. Visit the created page
4. Witness that you can't scroll between items in the spotlight/collection

## Context

We are trying to make Storm ADA compliant. This is required to do so!

**Comment by ryanbourneuk** [comment]
Resolved in v1.5.0
